media-libs/libgfx/libgfx-1.1.0.ebuild

```diff
@@ -10,9 +10,19 @@ SRC_URI="http://mgarland.org/dist/${P}.tar.gz"
 LICENSE="LGPL-2.1"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="static-libs"
+IUSE="static-libs test"
 
-PATCHES=( "${FILESDIR}"/${PV}-gcc4.3.patch )
+DEPEND="
+	virtual/opengl
+	x11-libs/fltk
+"
+
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PV}-gcc-4.3.patch
+	"${FILESDIR}"/${PV}-libPNG-1.0.6.patch
```

Comment:
This does not match the filename of the patch

Reply:
Thank you for catching that, what a silly mistake. One of my commits fixes that, and I tested it to make sure both patches are applied.